packages/garbo/src/resources/clanVIP.ts

```diff
@@ -0,0 +1,8 @@
+import { $item, Clan, get, have } from "libram";
+
+export function hotTubAvailable(): boolean {
+  return (
+    // FIXME getClanID returns a clan even if you aren't currently in one, as long as you were ever in a clan.
```

Comment:
I am extremely fine with this never getting fixed btw.

That being said, I'm a little surprised it's automatically coercing this to a boolean.